### DISCUSSION TITLE
Toshiba T300

### DISCUSSION THREAD

**Discussion by kheireddine-saidi** [keirf/flashfloppy#581] [Q&A]
I've been trying to fit a gotek drive flashed with the FlashFloppy firmware to a Toshiba T300 to no avail.

It uses DD floppies, i guess 40 tracks 8 sectors/track but i may be wrong, I didn't find much informations about this computer.

[http://www.opengate.at/blog/2019/09/toshiba-t300/](http://www.opengate.at/blog/2019/09/toshiba-t300/)

The original 5.25" drive have a 34 pin connector which was a surprise for me. Also I ignore the purpose of the two wires connector PJ3 (pointed with pink arrow) , any idea?.

I configured FF as an ibmpc interface, but i'm always getting error when trying to boot with gotek drive as second drive.

Any help would be appreciated.

Thank you.
![1637955072777](https://user-images.githubusercontent.com/30529067/143662508-85f90049-14a3-4dfd-b7cf-b8b81d4aaea0.jpg)
![1637955072820](https://user-images.githubusercontent.com/30529067/143662511-ecacb459-5ba4-4bd0-a38a-f59a28e3b115.jpg)
![1637955072862](https://user-images.githubusercontent.com/30529067/143662541-3c6d6be6-f435-48bb-8e12-b4b8099ad7df.jpg)
![1637955394449](https://user-images.githubusercontent.com/30529067/143662552-94941e47-b37a-4ebc-96ed-106ea0a1c627.jpg)

**Comment by keirf** [comment]
Could be all sorts of things. Do you know if your booting disk image is good?
